src/fiskaltrust.Middleware.Demo/RestPos.cs

```diff
@@ -0,0 +1,173 @@
+﻿using fiskaltrust.ifPOS.v0;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Demo.Helpers;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace fiskaltrust.Middleware.Demo
+{
+    public class RestPos : ifPOS.v1.IPOS
+    {
+        private string _url;
+        public RestPos(string url)
+        {
+            _url = url;
+        }
+
+        IAsyncResult ifPOS.v0.IPOS.BeginEcho(string message, AsyncCallback callback, object state)
+        {
+            throw new NotImplementedException();
+        }
+
+        IAsyncResult ifPOS.v0.IPOS.BeginJournal(long ftJournalType, long from, long to, AsyncCallback callback, object state)
+        {
+            throw new NotImplementedException();
+        }
+
+        IAsyncResult ifPOS.v0.IPOS.BeginSign(ifPOS.v0.ReceiptRequest data, AsyncCallback callback, object state)
+        {
+            throw new NotImplementedException();
+        }
+
+        string ifPOS.v0.IPOS.Echo(string message)
+        {
+            throw new NotImplementedException();
```

Comment:
We could call the `v0/echo` endpoint here. There is just a difference in the parameter, because the v0.Echo method takes a string as body.